failedpython.py

```diff
@@ -2,7 +2,7 @@
 
 import hashlib
 import os
-
+iport lib
```

Comment:
There is a typo in the import statement on line 5. It should be 'import lib' instead of 'iport lib'.